revolv/templates/project/project.html

```diff
@@ -1,240 +1,319 @@
+              </span> Days Left
+            </span>
           </div>
           <!-- end .blue-bar -->
           <div class="dark-blue-bar">
-            <span class="pull-left actual-energy"><span class="bold">{{ project.actual_energy }}</span> lbs CO<sub>2</sub> Avoided</span>
-            <span class="pull-right"><span class="bold">{{ donor_count }}</span> {{ settings.revolv_cms.ProjectPageSettings.donors_wording }}</span>
+            <span class="pull-left actual-energy">
+              <span class="bold">{{ project.actual_energy }}
+              </span> lbs CO
+              <sub>2
+              </sub> Avoided
+            </span>
+            <span class="pull-right">
+              <span class="bold">{{ donor_count }}
+              </span> {{ settings.revolv_cms.ProjectPageSettings.donors_wording }}
+            </span>
           </div>
           <!-- end .blue-bar -->
         </div>
         <!-- end .info-main -->
-        <div class="video-main">
-          <div class="video-area pull-left">
-            <div class="video">
-              {% if project.video_url %}
-                <iframe class="project_video youtube-player" allowfullscreen></iframe>
-              {% endif %}
-            </div>
-          </div>
-          <div class="txt-area">
-            <div class="txt-table">
-              <div class="txt-td font20">
-                <div class="bottom-bar mobile-bottom-bar">
-                  <ul class="social-ul">
-                    <li><a href="javascript:;" class="btn-icon-gray"><i class="icon-fb icons"></i></a></li>
-                    <li><a href="javascript:;" class="btn-icon-gray"><i class="icon-tw icons"></i></a></li>
-                    <li><a href="javascript:;" class="btn-icon-gray"><i class="icon-gg icons"></i></a></li>
-                  </ul>
-                  <span class="mobile-show share-txt">Share With Love.</span>
-                </div>
-                <p class="font20">
-                  {{ project.tagline }}
-                </p>
-                <p class="font14">
-                  {{ project.description|safe }}
-                </p>
-              </div>
-            </div>
-          </div>
-        </div>
         <!-- end .video-main -->
       </div>
     </div>
     <!-- end .info-section -->
   </div>
   <!-- end .details-active-project-module -->
-
-  <div class="project-updates-module">
+  <div class="col-sm-12 outer-basic-silver-gold project-updates-module">
     <div class="container">
-      <div class="main-area">
+      <div class="col-lg-8 col-md-8 col-sm-12 col-xs-12 outer-sil-gold">
+        <div class="col-sm-12">
+          <p class="content-gold-silver">{{ project.tagline }} 
+          </p>
+        </div>
+        <div class="col-sm-12">
+          <p class="content-gold-silver content_sec">{{ project.description|safe }}
+          </p>
+        </div>
+      </div>
+      <div class="col-lg-4 col-md-4 col-sm-12 col-xs-12 pull-right price-tables">
```

Comment:
This is where you need to use the django template tags from below instead of hard-coding this HTML.